### PR TITLE
Fixes #9138: rudder agent fails on aix

### PR DESCRIPTION
**Pull request by peckpeck**
https://www.rudder-project.org/redmine/issues/9138

### CODE REVIEW

File: share/commands/agent-run [hunk 1]
```diff
@@ -56,10 +56,11 @@ UUID=$(cat /opt/rudder/etc/uuid.hive 2>/dev/null)
 [ $? -ne 0 ] && UUID="Not yet configured"
 
 VERSION=`${RUDDER_BIN} agent version`
+AWK_FFLUSH=`awk 'BEGIN{fflush();}' /dev/null 2>/dev/null && echo OK`
```
Comment (amousset) [inline share/commands/agent-run]: Could you add a comment here to explain the issue?

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
Commit modified

**Comment by peckpeck** [comment]
OK, merging this PR

**Pull request merged**